installing/installing_ibm_z/upi/installing-ibm-z-lpar.adoc

```diff
@@ -83,12 +87,11 @@ include::modules/cluster-telemetry.adoc[leveloffset=+1]
 
 * link:https://access.redhat.com/solutions/4387261[How to generate SOSREPORT within {product-title} version 4 nodes without SSH]
 
-[id="next-steps_installing-ibm-z-lpar"]
-== Next steps
+* xref:../../../support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc#opting-out-remote-health-reporting_opting-out-remote-health-reporting[Opting out of remote health reporting]
 
-* xref:../../../machine_configuration/machine-configs-configure.adoc#rhcos-enabling-multipath-day-2_machine-configs-configure[Enabling multipathing with kernel arguments on {op-system}].
+[id="next-steps_ibm-z-lpar"]
```

Comment:
Not a merge stopping issue for this PR, but in a later PR please add a context to the end of this ID:
```suggestion
[id="next-steps_ibm-z-lpar_{context}"]
```